Tests/Schema.NET.Test/ValuesJsonConverterTest.cs

```diff
@@ -427,3 +547,42 @@ private class TestModel<T>
         }
     }
 }
+
+#pragma warning disable SA1649 // File name should match first type name
```

Comment:
Perhaps these two could be split into separate files in their own subfolder to avoid fighting the warnings.

Reply:
Yeah, happy to do that. I didn't want to initially as none of the other tests needed to do that - might put it into a folder named "Models" or something.